demo/src/Components/Cards/index.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+
+import Image from './Image'
+import Title from './Title'
+import Text from './Text'
+
+class Cards extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
```

Comment:
Don't need to use state here.  We can just have this component look a bit like this:
```
const StatusCards = () => (
  <div>
    <Card status="primary" />
    <Card status="success" />
  </div>
);
```

Should only use state for things that change within that component, here they are static and we can then avoid using `class` which is just :mask: LOL